WalletWasabi.Gui/UiConfig.cs

```diff
@@ -25,6 +25,7 @@ public class UiConfig : ConfigBase
 		private string? _lastSelectedWallet;
 		private string _windowState = "Normal";
 		private bool _runOnSystemStartup;
+		private bool _oobe = true;
```

Comment:
yes, so upgraders that do not have this property in their uiconfig.json will get the oobe.

Reply:
That is what `[DefaultValue(true)]` for above the property. But to make it work it should be configured like this
`[JsonProperty(PropertyName = "Oobe", DefaultValueHandling = DefaultValueHandling.Populate)]`
Instead of this:
`[JsonProperty(PropertyName = "Oobe")]`